Library/Aliases/git-hub

```diff
@@ -0,0 +1 @@
+../Formula/git-cmd-hub.rb
```

Comment:
In that case if we name it `git-hub-<something>` then it'll show up for `git-hub` searches. Maybe `git-hub-cmd` and drop the alias?